core/src/block_sync.rs

```diff
@@ -129,193 +101,140 @@ impl BlockSynchronizer {
         .send_to(self.broker.clone(), peer_id)
         .await;
     }
-
-    /// Continues the synchronization if it was ongoing. Should be called after `WSV` update.
-    #[iroha_futures::telemetry_future]
-    pub async fn continue_sync(&mut self) {
-        let (blocks, peer_id) = if let State::InProgress(blocks, peer_id) = self.state.clone() {
-            (blocks, peer_id)
-        } else {
-            return;
-        };
-
-        info!(blocks_left = blocks.len(), "Synchronizing blocks");
-
-        let (this_block, remaining_blocks) = if let Some((blck, blcks)) = blocks.split_first() {
-            (blck, blcks)
-        } else {
-            self.state = State::Idle;
-            self.request_latest_blocks_from_peer(peer_id).await;
-            return;
-        };
-
-        let mut network_topology = self.sumeragi.get_network_topology(&this_block.header());
-        // If it is genesis topology we cannot apply view changes as peers have custom order!
-        #[allow(clippy::expect_used)]
-        if !this_block.header().is_genesis() {
-            network_topology = network_topology
-                .into_builder()
-                .with_view_changes(this_block.header().view_change_proofs.clone())
-                .build()
-                .expect(
-                    "Unreachable as doing view changes on valid topology will not raise an error.",
-                );
-        }
-        if self.sumeragi.latest_block_hash() == this_block.header().previous_block_hash
-            && network_topology
-                .filter_signatures_by_roles(
-                    &[Role::ValidatingPeer, Role::Leader, Role::ProxyTail],
-                    this_block
-                        .verified_signatures()
-                        .map(SignatureOf::transmute_ref),
-                )
-                .len()
-                >= network_topology.min_votes_for_commit()
-        {
-            self.state = State::InProgress(remaining_blocks.to_vec(), peer_id);
-            /*
-            TODO: REPLACE
-            self.sumeragi
-                .do_send(CommitBlock(this_block.clone().into()))
-            .await;
-            */
-        } else {
-            warn!(block_hash = %this_block.hash(), "Failed to commit a block received via synchronization request - validation failed");
-            self.state = State::Idle;
-        }
-    }
 }
 
-/// The module for block synchronization related peer to peer messages.
-pub mod message {
-    use iroha_actor::broker::Broker;
-    use iroha_crypto::*;
-    use iroha_data_model::prelude::*;
-    use iroha_logger::prelude::*;
-    use iroha_macro::*;
-    use iroha_p2p::Post;
-    use iroha_version::prelude::*;
-    use parity_scale_codec::{Decode, Encode};
+use iroha_actor::broker::Broker;
```

Comment:
Please move the imports to the top, or use a wrapping module.

Reply:
has been done